pom.xml

```diff
@@ -10,16 +10,49 @@
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
     <groupId>com.leverx.springapp</groupId>
-    <artifactId>spring-app</artifactId>
-    <version>1.0.1</version>
+    <artifactId>demo</artifactId>
+    <version>1.0.2</version>
     <packaging>jar</packaging>
-    <name>Task1</name>
-    <description>Spring Boot app with Dockerfile</description>
+    <name>Task2</name>
+    <description>Spring boot app with docker-compose</description>
+
     <properties>
         <java.version>11</java.version>
     </properties>
 
     <dependencies>
+        <dependency>
```

Comment:
@blackavgn Please check ALL dependencies in pom.xml file. Do we really need them?

Reply:
Removed

Reply:
@blackavgn please comment it after changes has been pushed.